mmcv/ops/csrc/pytorch/bias_act.h

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
```

Comment:
Add header guards for all `.h` to avoid duplicate include. https://www.learncpp.com/cpp-tutorial/header-guards/

for example:
```cpp
#ifndef BIAS_ACT_H
#define BIAS_ACT_H

// your code

#endif // BIAS_ACT_H
```